src/calculateRentalCost.js

```diff
@@ -5,6 +5,13 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
Replace the hard-coded value `50` with a constant to represent the discount for renting 7 or more days. This adheres to the checklist's requirement to avoid using literals in expressions.